lib/report_results.py

```diff
@@ -547,7 +553,7 @@ def create_detailed_summary_uptake(summarised_data_dict,  formatted_latest_date,
         out_csv.to_csv(os.path.join(savepath["tables"], f"Cumulative vaccination figures among {group} population.csv"), index=True)
 
 
-def plot_dem_charts(summary_stats_results, cumulative_data_dict, formatted_latest_date, savepath, pop_subgroups=["80+", "70-79"], groups_dict=None, savepath_figure_csvs=None, include_overall=False, org_name="", suffix=""):
+def plot_dem_charts(summary_stats_results, cumulative_data_dict, formatted_latest_date, savepath, pop_subgroups=["80+", "70-79"], groups_dict=None, groups_to_exclude=None, savepath_figure_csvs=None, include_overall=False, org_name="", suffix=""):
```

Comment:
Is the `groups_to_exclude` in anticipation of using it in future or there to ensure defaults are correctly handled??

Reply:
This allows us to feed in the full list of demographic/clinical variables that were previously defined for each cohort (and are shown in tables) but exclude certain less interesting ones from being plotted as charts to save space.